content/en/continuous_integration/setup_tests/java.md

```diff
@@ -107,91 +97,62 @@ Configure the [Maven Surefire Plugin][3] and/or the [Maven Failsafe Plugin][4] t
 </plugin>
 {{< /code-block >}}
 
-Run your tests using the `ci-app` profile, for example:
+Run your tests as you normally do, specifying the environment where tests are being run (e.g. `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable. For example:
 
 {{< code-block lang="bash" >}}
-mvn clean verify -P ci-app
+DD_ENV=ci mvn clean verify -P ci-app
```

Comment:
Nit: At this point, I don't know if we should use `ci-visibility` instead of `ci-app`.

Reply:
Changed to `dd-civisibility`